packages/boards/sender/src/models/LightDto.cpp

```
@@ -0,0 +1,140 @@
+    }
+    if (color.length() == 8) {
+        return this->adjustColorToBrightness(color.substring(0, 6), brightness);
+    }
+
+    // brightness is max so no need to mention it
+    if (brightness == 255) {
+        return color;
+    }
+
+    char buff[2];
+    sprintf(buff, "%02x", brightness);
```

Cppcheck (2.18.0)

[error] Buffer is accessed out of bounds

(bufferAccessOutOfBounds)